ironic_bug_dashboard/configs/ironic.json

```diff
@@ -0,0 +1,83 @@
+{
+    "projects": [
+        "bifrost",
```

Comment:
need to add "networking-generic-switch" and "ironic-tempest-plugin"
also please use alphabetical order

Reply:
Noted.

Reply:
I think i-t-p might not have an LP yet. Should it?

Reply:
it probably should, even if it's just for internal use
I've noticed we do have bugs in itp but we don't really track them anywhere
even if it's just for "internal use" may be good to have them

Reply:
btw since this is causing issues I agree on commenting it out for the time being